sqlglot/transforms.py

```diff
@@ -9,6 +9,27 @@
     from sqlglot.generator import Generator
 
 
+def update_from(expression: exp.Expression) -> exp.Expression:
```

Comment:
We should rename this to reflect the intent, i.e. something like `update_from_to_merge_into`.